### DISCUSSION TITLE
Medusa V2 What is the event name for when admin click "Mark as shipped"?

### DISCUSSION THREAD

**Discussion by umarluqman** [medusajs/medusa#10766] [Q&A] [answered]
In the Medusa version 2 docs: https://docs.medusajs.com/resources/events-reference

There is no event reference for when admin mark the order as shipped.

<img width="1478" alt="Screenshot at Dec 30 10-52-50" src="https://github.com/user-attachments/assets/90036fd7-8e75-4291-a4ad-559a1d7e4f3c" />

The closest thing I could find it "order.fulfillment_created". Am I missing something?

**Comment by olivermrbl** [comment, marked as answer]
Thanks for highlighting. The event is `shipment.created` and should be in the reference.
